flows/index.md

```diff
@@ -0,0 +1,128 @@
+
+Flows integrate all of your project's segments (extractors, writer, transformations, other flows, etc.) by creating custom automated processes, chaining components to be run in a specific order, 
+and defining the execution schedule to **bring in the newest data available**. 
+
+First, you need to decide what exactly you want your flow to do, meaning **what tasks** should be executed. Then, you decide in **what order** you want them to run. 
+And finally, you will determine **when** you want the entire flow to be executed and **how often**. Notifications are available to help you monitor the entire process. 
+
+## What is the Flow Builder?
+Using the **Flow Builder** feature, you can create data flows by dragging and dropping the components together on a single screen.
+Even scheduling and automating your data pipelines takes just a few button clicks. 
+
+The flow builder enables you to do the following:
+
+- View your multi-step data pipeline in a single browser window, which is especially helpful when your data pipeline is complex. 
+- Create your flows with no data engineering skills.
+- Collaborate on the same data flows, set up the data pipeline as a team, and continue working on different components individually. The in-built version control lets your team see the changes others have made over time.
+- Build multiple data flows in a single view. Each flow is its own data pipeline.
+- Copy-and-paste an existing data flow to a new flow to reuse the work done across different data pipelines.
+
+Let us show you how to organize individual tasks into steps, set up notifications, and schedule the execution of your flow:
+
+- [Step 1: Access the Flow Builder](#access-the-flow-builder)
+- [Step 2: Build the flow](#build-the-flow)
+- [Step 3: Execute tasks in parallel](#execute-tasks-in-parallel)
+- [Step 4: Schedule and automate your flow](#schedule-and-automate)
+- [Step 5: Set up notifications](#set-up-notifications)
+- [Step 6: Check how your flow is running](#check-run-history)
+
+## Access the Flow Builder
+Select from the top menu **Flows -> Flows** and click the **Create Flow** button.
+   
+{: .image-popup}
+![Go to Flows](/tutorial/automate/automate1.png)
+
+Name your new flow and add an easy to understand description. Then click **Create Flow** again. 
+
+This will open up the **flow builder** view, where you can create your data flow.
+
+## Build the Flow
+Click **Select First Step** and start selecting the components that will bring in data from your selected data sources. Use the drop-down menu to select a particular configuration of the component.
+
+{: .image-popup}
+![Select First Step](/tutorial/automate/automate3.png)
+
+Use the plus icon to add other steps. 
+
+Apart from the extractor(s), you'll want to add selected transformations and writers that will load your transformed data into your selected data destination.
```

Comment:
Tady je možná trošku skok "Apart from the extractor" jaký extractory? (btw, nemá to teda bejt data sources?). Možná by to chtělo něco jako "typically the flow starts with extractors, or if all tables you need are already in the the project, start with transformations..."

Reply:
Supr, to je lepsi. Dik

Ad ex/wr vs data sources... – uz jakou dobu je tak nejak oficialne rozhodnuto (Chocho), ze se vracime zpatky. Ale porad nebylo zadny oficialni oznameni napric firmou. Ted to je na Vladovi, kterej to bere znovu zpatky na produkt, jestli je to tak opravdu s konecnou platnosti. 
Zeptam se ho pak a kdyztak to znovu predelam na data sources...